UPGRADE_GUIDE.md

```diff
@@ -13,7 +13,7 @@ Change your `mix.exs`:
 def deps do
   [
     {:goth, "~> 1.3-rc"},
-    {:hackney, "~> 1.17"}
+    {:finch, "~> 0.9.0 or ~> 0.10.0 or ~> 0.11.0 or ~> 0.12.0"}
```

Comment:
We're close to Finch v1.0 and it is unlikely to introduce breaking changes so it's fine to have broader requirement. I'll do the same for Req later.

```suggestion
    {:finch, "~> 0.9"}
```